pkg/cmd/controller/authorization.go

```diff
@@ -1,18 +1,44 @@
 package controller
 
-import "github.com/openshift/openshift-controller-manager/pkg/authorization/defaultrolebindings"
+import (
+	"github.com/openshift/openshift-controller-manager/pkg/authorization/defaultrolebindings"
+	"k8s.io/client-go/kubernetes"
+)
 
 func RunDefaultRoleBindingController(ctx *ControllerContext) (bool, error) {
 	kubeClient, err := ctx.ClientBuilder.Client(infraDefaultRoleBindingsControllerServiceAccountName)
 	if err != nil {
 		return true, err
 	}
 
-	go defaultrolebindings.NewDefaultRoleBindingsController(
-		ctx.KubernetesInformers.Rbac().V1().RoleBindings(),
-		ctx.KubernetesInformers.Core().V1().Namespaces(),
+	return runRoleBindingController(ctx, kubeClient, "DefaultRoleBindingController")
+}
+
+func RunBuilderRoleBindingController(ctx *ControllerContext) (bool, error) {
+	kubeClient, err := ctx.ClientBuilder.Client(infraBuilderRoleBindingsControllerServiceAccountName)
```

Comment:
Keep the service account name here for now. Refactoring how this done adds a significant amount of scope, and is not critical to address right now. I'd add the following comment:

```suggestion
        // Role binding controllers currently share the same service account, as these are created by "boostrap" logic located elsewhere in Openshift
        // TODO: Refactor the controller service accounts to be managed by openshift-controller-manager-operator
	kubeClient, err := ctx.ClientBuilder.Client(infraDefaultRoleBindingsControllerServiceAccountName)
```

Reply:
cc @deads2k - I seem to recall we did this kind of refactoring a long time ago for the `route` controllers (which eventually spun out of openshift-controller-manager).